indico/modules/events/abstracts/operations.py

```diff
@@ -271,22 +325,57 @@ def create_abstract_review(abstract, track, user, review_data, questions_data):
         review.ratings.append(AbstractReviewRating(question=question, value=value))
     db.session.flush()
     logger.info("Abstract %s received a review by %s for track %s", abstract, user, track)
+    log_data = {
+        'Track': track.title,
+        'Action': orig_string(review.proposed_action.title),
+        'Comment': review.comment
+    }
+    if review.proposed_action == AbstractAction.accept:
+        log_data['Contribution type'] = (review.proposed_contribution_type.name if review.proposed_contribution_type
+                                         else None)
+    elif review.proposed_action == AbstractAction.change_tracks:
+        log_data['Other tracks'] = sorted(t.title for t in review.proposed_tracks)
+    elif review.proposed_action in {AbstractAction.mark_as_duplicate, AbstractAction.merge}:
+        log_data['Other abstract'] = '#{0.friendly_id} ({0.title})'.format(review.proposed_related_abstract)
```

Comment:
You can even do `{0['foo']}` for dicts. ;)
I usually prefer passing separate arguments but here I did it mostly to avoid having to wrap it.

FWIW, I'm considering to add a `verbose_title` property to `Abstract` returning this...